lib/iso-execute-server.js

```diff
@@ -26,7 +26,7 @@ var middleware = function (req, res) {
     execute.apply(req, [req.params.name].concat(req.body)).then(function (R) {
         res.send({rpc: R});
     })['catch'](function (E) {
-        console.warn(E.stack);
+        console.warn((E.stack || E));
```

Comment:
It will be better to show more detail about the error:

```javascript
console.warn('Error when do server side iso-call.execute() :');
console.warn(E.stack || E);
```